orderBook.js

```diff
@@ -0,0 +1,51 @@
+function isOrderMatch (incomingOrder, existingOrder)
+{
+  if (incomingOrder.price === existingOrder.price &&
+    incomingOrder.type !== existingOrder.type)
+  {
+    return true
+  }
+  return false
```

Comment:
This if is entirely unnecessary, you can just return the condition itself since you are returning true when the condition is true. The only time you should ever have a `return true` or `return false` in an if statement is when you are using the return to stop a loop.

```JavaScript
function isOrderMatch (incomingOrder, existingOrder) {
  return incomingOrder.price === existingOrder.price &&
    incomingOrder.type !== existingOrder.type
}
```